app/src/main/java/com/example/epamandroid/MainActivity.kt

```diff
@@ -1,12 +1,21 @@
 package com.example.epamandroid
 
+import android.content.Intent
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_main.button1
+import kotlinx.android.synthetic.main.activity_main.second_activity_button
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        button1.setOnClickListener {
```

Comment:
what does `button1` do? Please give a more meaningful name.

Reply:
@egorikftp Oops, this is a useless button, I forgot to delete it